docs/technical-reference/classes/ERC725.md

```diff
@@ -6,8 +6,8 @@ This package is currently in early stages of development, <br/>use only for test
 
 ## Type parameters
 
-| Name     | Type                    | Description                                |
-| :------- | :---------------------- | :----------------------------------------- |
+| Name | Type | Description |
+| :------ | :------ | :------ |
```

Comment:
It's not important but it would be nice if we can automatically run prettier on the .md files too (i did it manually the last time, hence the diff, sorry 😄 )

I'm not saying this should be done now, just saying it would be nice 🥳

Reply:
Indeed